### PR TITLE
profile menu collapsable

### PR DESCRIPTION
**Pull request by TheMaja**
- profile click dropdown menu
- dropdown menu working
- mobile menu updated

Related issues: [please specify]

## Description:

What are you up to? Fill us in :)

## Screenshots/videos

Uploading dropdownmenu.mp4…

https://github.com/Umuzi-org/Tilde/assets/73611175/c8d30ab6-7923-4a1a-9f4b-828c9dc6f49e

## I solemnly swear that:

- [x] My code follows the style guidelines of this project
- [ ] I have merged the develop branch into my branch and fixed any merge conflicts
- [x] I have performed a self-review of my own code
- [ ] I have commented my code in hard-to-understand areas
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have tested new or existing tests and made sure that they pass

### CODE REVIEW

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
On the right track Themba...just a few things

- I like how smooth the opening animation is :slightly_smiling_face: 
- The menu opens the wrong way(to the right) on larger screens causing the website to break/go out of bounds. Please just use original position( where it opens to the left) and  make sure clicking the profile icon closes or opens it.

image
> original

image
> current 

- The way the menu shows on the smaller screens got slightly worse. Please stick with the original layout

image
> original

image
> current

- Remember to make sure that clicking the hamburger icon still closes or opens the menu on the smaller screens.

**Review by Ngoakor12 — CHANGES_REQUESTED** [review]
Mostly good Themba,

One tiny thing...
The hover background should have matching rounded corners when you're hovering or not hovering on the first and last option in the dropdown menu.

E.g. The top left and right should have a border-radius so that the corners are rounded. 
image

**Review by Ngoakor12 — APPROVED** [review]

File: backend/frontend/templates/frontend/base_logged_in.html [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {% extends 'frontend/base.html' %}
 {% load static %}
 {% block main_head %}
+    <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet" />
```
Comment (sheenarbw) [inline backend/frontend/templates/frontend/base_logged_in.html]: frontend/templates/frontend/base.html includes the line:

```
    <script src="https://cdn.tailwindcss.com"></script>
```

And this template extends base.

Why are you including tailwind again? And why should different parts of the website run different versions of tailwind?

**Review by sheenarbw — CHANGES_REQUESTED** [review]
Just one thing I could find

**Review by sheenarbw — APPROVED** [review]

**Pull request merged**